tests/api2/test_435_smb_registry.py

```diff
@@ -70,7 +70,7 @@ def create_smb_share(path, share_name, mkdir=False, options=None):
     cr_opts = options or {}
 
     if mkdir:
-        call('filesystem.mkdir', path)
+        call('filesystem.mkdir', {'path': path, 'options': {'raise_chmod_error':   False}})
```

Comment:
Flake8: too many spaces before `False`
Same with the other two below.

Also... 
Why is it now necessary to pass `{'raise_chmod_error': False}`  ?

Reply:
Because SMB datasets have a restricted ZFS aclmode.